priv/static/js/beaker-frontend.js

```diff
@@ -0,0 +1 @@
NiIgc3R5bGU9ImZpbGw6I2ZmZjtzdHJva2Utd2lkdGg6MTdweDtzdHJva2U6IzAwMDsiLz48L2c+PGc+PHBhdGggZD0iTTI1Ny42MDgsMjA4LjE1N2M1MC4xMzIsLTI2LjcyOSA3MS4yNzUsMTcuMzQ3IDEyNS4yODQsMTYuOWM1NS4xNDEsLTAuNDU3IDQ3LjM3MSwtMjEuMjMyIDEwNi40NjUsNS4xMDZsMTg1Ljg1LDMyMS43MDlsLTYwNi40MTQsMGwxODguODE1LC0zNDMuNzE1WiIgc3R5bGU9ImZpbGw6IzY4NTI3MzsiLz48cGF0aCBkPSJNMzQ5LjM5MiwyNDkuNzVjNTAuMzMzLDAuMzIyIDkwLjkzNiw0MS40NDggOTAuNjEzLDkxLjc4MmMtMC4zMjIsNTAuMzMzIC00MS40NDgsOTAuOTM2IC05MS43ODIsOTAuNjEzYy01MC4zMzMsLTAuMzIyIC05MC45MzYsLTQxLjQ0OCAtOTAuNjEzLC05MS43ODJjMC4zMjIsLTUwLjMzMyA0MS40NDgsLTkwLjkzNiA5MS43ODIsLTkwLjYxM1oiIHN0eWxlPSJmaWxsOiNmZmY7Ii8+PHBhdGggZD0iTTI0NC44NTQsNDA3LjgzN2MzNC41MDYsMC4yMjEgNjIuMzQsMjguNDE0IDYyLjExOSw2Mi45MTljLTAuMjIxLDM0LjUwNiAtMjguNDE0LDYyLjM0IC02Mi45Miw2Mi4xMTljLTM0LjUwNSwtMC4yMjEgLTYyLjM0LC0yOC40MTQgLTYyLjExOCwtNjIuOTJjMC4yMjEsLTM0LjUwNSAyOC40MTQsLTYyLjM0IDYyLjkxOSwtNjIuMTE4WiIgc3R5bGU9ImZpbGw6I2ZmZjsiLz48cGF0aCBkPSJNNjM0LjQ1LDQ2Ni43MzNsLTIxNy45NzgsLTAuMzUzbC0wLjA1OSwxOS43MWwyMTcuOTc4LDAuMzUzbDAuMDU5LC0xOS43MVoiIHN0eWxlPSJmaWxsOiMwMDA7ZmlsbC1vcGFjaXR5OjAuNjc0NTE7Ii8+PHBhdGggZD0iTTU5Ny4zNTksNDA5LjAzOWwtMTgwLjg4NywtMC4yOTNsLTAuMDU3LDE4Ljk4MWwxODAuODg3LDAuMjkzbDAuMDU3LC0xOC45ODFaIiBzdHlsZT0iZmlsbDojMDAwO2ZpbGwtb3BhY2l0eTowLjY3NDUxOyIvPjxwYXRoIGQ9Ik01NjIuMywzNDQuNjNsLTE0NS44MjgsLTAuMjM2bC0wLjA1OSwxOS45MzFsMTQ1LjgyOCwwLjIzN2wwLjA1OSwtMTkuOTMyWiIgc3R5bGU9ImZpbGw6IzAwMDtmaWxsLW9wYWNpdHk6MC42NzQ1MTsiLz48cGF0aCBkPSJNNTQ2Ljk0MywyNy44YzkuOTU1LDAuMDAxIDE4LjAzNiw4LjA4MiAxOC4wMzYsMTguMDM2bDAsMzYuMDczYzAsOS45NTQgLTguMDgyLDE4LjAzNiAtMTguMDM2LDE4LjAzNmwtODUuOTc5LDBjMC4xMjMsMS4zNDUgMC4xODYsMi43MDggMC4xODYsNC4wODZsMCw3Mi4zMjNsMjE0LjA1NywzNzUuNTE4bC02MDYuNDE0LDBsMjE0LjA1NywtMzc1LjUxOGwwLC03Mi4zMjNjMCwtMS4zNzggMC4wNjMsLTIuNzQxIDAuMTg2LC00LjA4NmwtODUuOTc5LDBjLTkuOTU0LDAgLTE4LjAzNiwtOC4wODIgLTE4LjAzNiwtMTguMDM2bDAsLTM2LjA3M2MwLC05Ljk1NCA4LjA4MSwtMTguMDM1IDE4LjAzNiwtMTguMDM2bDM0OS44ODYsMFoiIHN0eWxlPSJmaWxsOm5vbmU7c3Ryb2tlLXdpZHRoOjMycHg7c3Ryb2tlOiMwMDA7Ii8+PHBhdGggZD0iTTI3OC4zNTQsOTIuNDgzbDE5My4wOTksMC4wMjYiIHN0eWxlPSJmaWxsOm5vbmU7c3Ryb2tlLXdpZHRoOjE3cHg7c3Ryb2tlOiMwMDA7Ii8+PC9nPjwvZz48L2c+PHRleHQgeD0iLTE2LjMzNnB4IiB5PSI3MzUuMTAzcHgiIHN0eWxlPSJmb250LWZhbWlseTonT3BlblNhbnMtRXh0cmFib2xkJztmb250LXNpemU6MjA2LjE4N3B4O2ZpbGw6I2ZmZjtzdHJva2Utd2lkdGg6MXB4O3N0cm9rZTojMDAwOyI+Qjx0c3BhbiB4PSIxMjguNDUxcHggMjYxLjI3N3B4IDM5NS4xNTJweCA1MzIuNDlweCA2NjUuMzE2cHggIiB5PSI3MzUuMTAzcHggNzM1LjEwM3B4IDczNS4xMDNweCA3MzUuMTAzcHggNzM1LjEwM3B4ICI+ZWFrZXI8L3RzcGFuPjwvdGV4dD48L2c+PC9zdmc+"),e.appendChild(t,n),t},buildRenderNodes:function(){return[]},statements:[],locals:[],templates:[]}}())}),define("beaker-frontend/config/environment",["ember"],function(e){var t="beaker-frontend";try{var n=t+"/config/environment",a=e["default"].$('meta[name="'+n+'"]').attr("content"),r=JSON.parse(unescape(a));return{"default":r}}catch(i){throw new Error('Could not read config from meta tag with name "'+n+'".')}}),runningTests||require("beaker-frontend/app")["default"].create({name:"beaker-frontend",version:"0.0.0+f9f2cc14"});
```

Comment:
hahaha, wonder what we should do about that, and if it'd keep warning everytime we do something...